src/elements/emby-textarea/emby-textarea.js

```diff
@@ -4,6 +4,17 @@ import '../emby-input/emby-input';
 
 /* eslint-disable indent */
 
+    function calculateOffset(textarea) {
```

Comment:
Since this is being moved, it should go above the eslint comment above so its indentation can be checked.